locksmith/src/controllers/v2/priceController.ts

```diff
@@ -15,6 +18,17 @@ export const amount: RequestHandler = async (request, response) => {
     amount,
     address,
   })
+
+  if (lockAddress && ethers.utils.isAddress(lockAddress)) {
+    result.creditCardEnabled = await getCreditCardEnabledStatus({
+      lockAddress: Normalizer.ethereumAddress(lockAddress),
+      network,
+      totalPriceInCents: result.priceInAmount ?? 0,
+    })
+  } else {
+    result.creditCardEnabled = false
+  }
```

Comment:
should this not just be a separate endpoint to check if credit card is enabled?

Reply:
added new endpoint